### PR TITLE
Add workflow dispatch trigger

### PR DESCRIPTION
**Pull request by jindraregal**

### CODE REVIEW

**Comment by gyfis** [comment]
@jindraregal I'd keep rails 6.2, what's up with that?

**Comment by jindraregal** [comment]
> @jindraregal I'd keep rails 6.2, what's up with that?

It seemed to me there is no Rails 6.2, it had a hard time fetching it. Or does it exist?

**Comment by gyfis** [comment]
@jindraregal haha you're right, apologies! This confused me but it really seems Rails 6.2 is not a thing. Sorry about that!
image

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,12 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
+
+gem 'rails', '~> 6.0.0'
```
Comment (gyfis) [inline Gemfile]: I wouldn't add this since we have the `rails-6.0.gemfile`

**Comment by curusarn** [comment]
Rebased and resolved a comment.

**Review by gyfis — APPROVED** [review]
❤️

**Comment by PetrHeinz** [comment]
Closing this PR as all main points were addressed in other PRs (#16, #17, #18)

**Pull request closed without merging**